src/components/screens/main-screen.tsx

```diff
@@ -702,17 +667,78 @@ class MainScreen extends React.Component<Props, State> {
   };
 
   /**
-   * Returnes a route line represented by array of coordinates 
-   *
-   * @param defaultRoutingMode default routing mode data
-   * @returns array of coordinates
+   * Creates a route list offset
+   * 
+   * @param routeLine the route line as array of points
+   * @returns modified route line
```

Comment:
```suggestion
   * Creates a route offset
   * 
   * @param routeAlts the route alternatives
   * @returns modified route lines
```
Should it be like that?